### DISCUSSION TITLE
detection with different source

### DISCUSSION THREAD

**Discussion by jehan88** [ultralytics/yolov5#8877] [Q&A]
hello all,
am using detection.py but I have error I tried to fix it but noway, the  image (folder then folder1 then pano then img) so I need to enter inside each folder then folder then image.

import os
import glob
pred_path = ('/content/gdrive/MyDrive/Projects/contern/')  
for folder in os.listdir(pred_path) :
  for sub_folder in os.listdir(pred_path + folder):
       img_path =pred_path + sub_folder + '/pano' 
      !python detect.py --weights /content/gdrive/MyDrive/Object_detection/yolov55/runs/train/exp15/weights/last.pt --img 416 --conf 0.4 --source img_path  --save-txt --save-crop --save-conf  --project /content/gdrive/MyDrive/Object_detection/project_test/contern/ --name 'contern-crop'   
    

the error(File "/content/gdrive/MyDrive/Object_detection/yolov55/utils/dataloaders.py", line 187, in __init__
    raise Exception(f'ERROR: {p} does not exist')
Exception: ERROR: /content/gdrive/MyDrive/Object_detection/yolov55/img_path does not exist)

**Comment by glenn-jocher** [comment]
@jehan88 user error on your part. You've passed a --source path that does not exist.
